fedimint-client/src/module/mod.rs

```diff
@@ -142,6 +146,10 @@ where
             .await
     }
 
+    pub fn decoders(&self) -> ModuleDecoderRegistry {
+        self.client.client.get().decoders().clone()
+    }
```

Comment:
Are the actually cheap to clone and do we need them to be owned? Iirc this is a `BTreeMap<_, Arc<_>>`, so not great/not terrible.

Reply:
Cheap enough unless proved otherwise. :)